urbanairship-accengage/src/main/java/com/urbanairship/accengage/notifications/AccengageNotificationProvider.java

```diff
@@ -21,11 +23,16 @@
  * Accengage notification provider.
  */
 public class AccengageNotificationProvider implements NotificationProvider {
+    private final AirshipConfigOptions configOptions;
+
+    public AccengageNotificationProvider(AirshipConfigOptions configOptions) {
```

Comment:
breaking change as well. Might be better to make this lookup the config options from the singleton for now `UAirship.shared().getAirshipConfigOptions()` if the color or icon are null below